src/cmd_handlers.c

```diff
@@ -1011,6 +1016,44 @@ goto_cmd(const cmd_info_t *cmd_info)
 	return 0;
 }
 
+static int
+regedit_cmd(const cmd_info_t *cmd_info)
+{
+	if (cmd_info->argc > 1)
+		return CMDS_ERR_TRAILING_CHARS;
+
```

Comment:
```suggestion
```

Because of `.min_args = 0,   .max_args = 1`, `argc` will always be either `0` or `1`, so this if-statement is unnecessary.